pkg/executables/kubectl.go

```diff
@@ -735,3 +735,35 @@ func (k *Kubectl) ValidateNodesVersion(ctx context.Context, kubeconfig string, k
 	}
 	return nil
 }
+
+func (k *Kubectl) ApplyTaint(ctx context.Context, kubeconfig string, node string, taint corev1.Taint) error {
+	params := []string{"taint", "nodes", node, fmt.Sprintf("%s=%s:%s", taint.Key, taint.Value, taint.Effect), "--kubeconfig", kubeconfig}
+	_, err := k.executable.Execute(ctx, params...)
+	return err
+}
+
+func (k *Kubectl) GetWorkerNodes(ctx context.Context, kubeconfig string) ([]string, error) {
+	var workerNodes []string
+	template := "{{range .items}}{{.metadata.name}}\n{{end}}"
+	params := []string{"get", "nodes", "-o", "go-template", "--template", template, "--kubeconfig", kubeconfig}
```

Comment:
wouldn't be easier to just to just get all nodes with `get nodes -o yaml` and then just iterate over them to do the filtering? Instead of using one command per node
Also couldn't we just do the filtering using kubectl?